libs/designsystem/src/lib/components/segmented-control/segmented-control.component.spec.ts

```diff
@@ -142,4 +142,40 @@ describe('SegmentedControlComponent', () => {
       expect(component.value).toBe(items[2]);
     });
   });
+
+  describe("in 'compact chip' mode", () => {
```

Comment:
Consider combining this in a loop with the above describe block, as they are virtually identical.